config/sync/block.block.views_block__eif_related_entities_block_1.yml

```diff
@@ -0,0 +1,30 @@
+uuid: 287cfd6f-101c-4215-95b4-1e7bb79d3a3e
+langcode: en
+status: true
+dependencies:
+  config:
+    - views.view.eif_related_entities
+  module:
+    - joinup_core
```

Comment:
Why does this depend on `joinup_core`? Normally when this is here we are using one of the formatters from joinup_core but this doesn't seem to be the case now. Was this added automatically during the config export?

Reply:
Yes this was automatic. I am not sure why this is. I am going to remove it and test the configuration. Otherwise, I am going to leave it.